Playmate/View Controllers/Home Tab/UpcomingSessionsViewController.h

```diff
@@ -0,0 +1,17 @@
+//
+//  UpcomingSessionsViewController.h
+//  Playmate
+//
+//  Created by Jocelyn Tseng on 7/18/22.
+//
+
+#import <UIKit/UIKit.h>
+#import <Parse/Parse.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UpcomingSessionsViewController : UIViewController
```

Comment:
I think, if you add here (remove it from the .m)
you will not have to cast to (id)